src/components/process-menu/process-menu.tsx

```diff
@@ -61,6 +62,12 @@ interface ProcessMenuProps {
 
 export const ProcessMenu = ({ id, targetType }: ProcessMenuProps) => {
   const hasEnhancedProcessMenu = useFeatureToggle("MMH.EnhancedProcessMenu");
+  const enableChangeOfName = useFeatureToggle("MMH.EnableChangeOfName");
+
+  const processEnableStatus: Record<string, boolean> = {
+    soletojoint: true,
+    changeOfName: enableChangeOfName,
+  };
```

Comment:
Feature toggles should only be set in configuration files, I'd suggest to have a feature toggle per process, so for example something like this:

```
  const processEnableStatus: Record<string, boolean> = {
    soletojoint: useFeatureToggle("MMH.SoleToJoint"),
    changeOfName: useFeatureToggle("MMH.ChangeOfName"),
  };
```